grails-app/services/streama/MarshallerService.groovy

```diff
@@ -303,67 +312,6 @@ class MarshallerService {
         returnArray['backdrop_path'] = movie.backdrop_path
         returnArray['poster_path'] = movie.poster_path
         returnArray['trailerKey'] = movie.trailerKey
-        returnArray['tags'] = movie.tags
```

Comment:
What happened here? you created a whole new marshaller? And you removed a bunch of my marshallers? Why did you do this?  IF you do something this extreme, your commit messages should be much much much more verbose!